src/Storage/Commands.Storage/File/Cmdlet/GetAzureStorageShare.cs

```diff
@@ -39,6 +40,14 @@ public class GetAzureStorageShare : AzureStorageFileCmdletBase
         public string Prefix { get; set; }
 
         [Parameter(
+        Position = 1,
+        Mandatory = false,
+        ParameterSetName = Constants.SpecificParameterSetName,
+        HelpMessage = "SnapshotTime of the file share snapshot to be gotten.")]
```

Comment:
@blueww same comment about changing "gotten" to "received"

Reply:
Updated